### PR TITLE
adding 512 release notes. Waiting on additional info for 2 issues.

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.1.md [hunk 1]
```diff
@@ -3,6 +3,72 @@ title: 128T 5.1 Release Notes
...
+------
+- **I95-39580 Reference Candidate Config Values in a Template:** Using the `refconfig` tag in Advanced Mode allows you to reference a value from a running configuration. 
+------
+- **I95-39587 Duplicate entries in show commands on PCLI when using bulk-edit:** Resolved an issue with the use of bulk-edit in the PCLI.
+------
+- **I95-39602 Show peer path status under `show peers hostnames`:** Replaced redundant Router column with `Node`, and added a `Status` column.
+------
+- **I95-39632 Power Saver using stale data:** Resolved an issue with the power saver using stale data after the device-type changes between locked and lockless. 
+------
+- **I95-39639 Packet transmit fails when traffic-eng is enabled with multiple worker cores:**
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:43]: did you mean to add some more detailed text to follow the heading, such as "this issue has now been resolved"

File: docs/release_notes_128t_5.1.md [hunk 2]
```diff
@@ -3,6 +3,72 @@ title: 128T 5.1 Release Notes
...
+------
+- **I95-39641 `show device-interface extended-stats` command reported unavailable when the virtio device does not expose extended stats:** Resolved an issue with the show device-interface extended-statistics/registers command.
+------
+- **I95-39695 Delete Session feature not working:** Resolved an issue preventing this from working.
+------
+- **I95-39698 Alibaba Cloud with enhanced networking sometimes doesn't work:** Resolved an issue between Virtio and RSS causing an unbalanced queue.
+------
+- **I95- Remote router login ignores 'User' selection:**
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:51]: did you mean to add some more detailed text to follow the heading, such as "this issue has now been resolved"

File: docs/release_notes_128t_5.1.md [hunk 3]
```diff
@@ -3,6 +3,72 @@ title: 128T 5.1 Release Notes
...
+------
+- **I95-39711 PCLI unhandled error when exporting a config using a name that has already been used.** The PCLI now provides a clear error message describing the issue.
+------
+- **I95-39764 Per Adjacency Traffic Eng Crashing For Multiple Paths on Configuration:** Resolved an issue when receiving adjacencies with out of order path-indexes.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:55]: Eng should be expanded to engineering

File: docs/release_notes_128t_5.1.md [hunk 4]
```diff
@@ -3,6 +3,72 @@ title: 128T 5.1 Release Notes
...
+------
+- **I95-39711 PCLI unhandled error when exporting a config using a name that has already been used.** The PCLI now provides a clear error message describing the issue.
+------
+- **I95-39764 Per Adjacency Traffic Eng Crashing For Multiple Paths on Configuration:** Resolved an issue when receiving adjacencies with out of order path-indexes. 
+------
+- **I95-39782 The aggregate stats pull from highway logs provides more detail than necessary:** Reduced the unnecessary detail. 
+------
+- **I95-39796 Conductor and Authority missing from GUI on first login:** Added multiple retries to retrieve system data upon first login. 
+------
+- **I95-39798 Token update and available version update stuck on GPG key access:** The upgrade check has been modified to verify the token update.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:61]: See my comments from 4.5.8 release notes

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**